src/editor/src/DebugTooltip.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "IEditorPanel.hpp"
+#include <glm/vec3.hpp>
+
+namespace Hush {
+	class DebugTooltip final : public IEditorPanel {
+	public:
+		static inline DebugTooltip* s_debugTooltip = nullptr;
```

Comment:
I have one question, why a static variable?

Reply:
Left this for prototyping, not meant for production